docs/startup/preview/index.md

```diff
@@ -0,0 +1,529 @@
+
+#### External Partition Management
+With external partition management, Samza doesn’t manage the partitioning by itself. Instead it uses a `PassthroughJobCoordinator` which honors whatever mapping is provided by the [SystemStreamPartitionGrouper](/learn/documentation/{{site.version}}/api/javadocs/org/apache/samza/container/grouper/stream/SystemStreamPartitionGrouper.html). There are two common patterns for external partition management:
+
+* **Using high level Kafka consumer** - partition assignment is done by the high level Kafka consumer itself. To use this model, you need to implement and configure a SystemFactory which provides the Kafka high level consumer. Then you need to configure *job.systemstreampartition.grouper.factory* to *org.apache.samza.container.grouper.stream.AllSspToSingleTaskGrouper* so Kafka's partition assignments all go to one task.
+* **Customized partitioning** - partition assignment is done with a custom grouper. The grouper logic is completely up to you. A practical example of this model is to implement a custom grouper which reads static partition assignments from the configuration.
+
+Samza ships with a `PassthroughJobCoordinatorFactory` which facilitates this type of partition management.
+
+#### Dynamic Partition Management
+
+With, dynamic partitioning, partitions are distributed between the available processors at runtime. If the number of available processors changes (some got shut down, or added) the mapping will be regenerated and re-distributed to all the processors.
+Information about current mapping is contained in a special structure called JobModel.
+There is one leader processor which generates the JobModel and distributes it to the other processors. The leader is determined by a “leader election” process.
+
+Let’s take a closer look at how dynamic coordination works.
+
+#### Coordination service
+
+Dynamic coordination of the processors assumes presence of a coordination service. The main responsibilities of the service are:
+
+* **Leader Election** - electing a single processor, which will be responsible for JobModel calculation and distribution or for intermediate streams creation.
+* **Central barrier and latch** - coordination primitives used by the processors.
+* **JobModel notifications** - notifying the processors about availability of a new JobModel.
+* **JobModel storage** the coordination service dictates where the JobModel is persisted.
+
+The coordination service is currently derived from the job coordinator factory. Samza ships with a `ZkJobCoordinatorFactory` implementation which has a corresponding `ZkCorrdinationServiceFactory`.
+
+Let’s walk through the coordination sequence for a Zookeeper based embedded application:
+
+* Each processor (participant) will register with the pluggable coordination service. During the registration it will provide its own participantId.
+* One of the participants will be elected as the Leader.
```

Comment:
> Leader

Throughout this section, "Leader" is capitalized. This might not be right, unless "Leader" is referring to a specific class or something (I don't think it is).